.travis.yml

```diff
@@ -38,7 +36,8 @@ addons:
 
 before_install:
   - rm -f Gemfile.lock
-  - gem install -v '~> 1.16' bundler
+  - for x in ${HOME}/.rvm/gems/*; do gem uninstall -I -x -i "${x}" -v '>= 1.17' bundler || true; gem uninstall -I -x -i "${x}@global" -v '>= 1.17' bundler || true; done
```

Comment:
When I ran this on my workstation, it stopped and waited for input because bundler version 1.17.1 and 1.17.3 were installed.  Should there be a --force option on this or do we not think it will ever have 2 versions of the bundler installed because it is a clean build?